### PR TITLE
Remove test code/examples and disable the inbox activity panel.

### LINKED ISSUE #341: Activity Panel: Hide Inbox Panel
Another quick issue in the series of tidying up the plugin for a release - I'm thinking we should hide the Inbox Activity Panel for now. Since this feature is the furthest from being in a somewhat usable state ( blocked by development of the Notif APIs within core ), it seems that we could remove it from the view until we are ready to actively work on this feature.

Let's not remove any of the underlying supporting code, just remove the tab.

### PR DESCRIPTION
**Pull request by emdashcodes**
This should be easy to review since it's 99% 🔴changes 😄. Fixes #341, #340, #339.

This PR removes some of the test/examples we've accumulated, now that we have charts and summary numbers on other pages. It also removes the inbox tab since we are a ways off from being able to support it.

Note: It seems like the positioning on the activity panel tabs is a little off. I'm guessing maybe a regression from the IE11 fix? cc @Aljullu. I didn't fix it here because I'm out of the house today and don't have my Windows machine on me to make sure I don't rebreak it there -- but I'll fix it later if you don't get to it first.

To Test:
* Check in the activity panel and verify the inbox tab is gone. Shrink your screen down to mobile and open the activity panel. Make sure it defaults to orders.
* Check the dashboard and verify test elements are gone.
* Check the analytics navigation and make sure the test report is gone.

### CODE REVIEW

**Review by ryelle — APPROVED** [review]
Looks good, thanks for cleaning up. I did notice the "Overview" page still has the search box demo on it, I don't know if you want to remove that too - it's also an example.

Pre-approving if you want to merge this in, but if you want to remove the search I can check again 🙂

**Comment by Aljullu** [comment]
> Note: It seems like the positioning on the activity panel tabs is a little off. I'm guessing maybe a regression from the IE11 fix? cc @Aljullu. I didn't fix it here because I'm out of the house today and don't have my Windows machine on me to make sure I don't rebreak it there -- but I'll fix it later if you don't get to it first.

I might have broken something. You can leave that one to me, I will take a look tomorrow. :slightly_smiling_face:

File: client/layout/activity-panel/index.js [hunk 1]
```diff
@@ -91,17 +91,11 @@ class ActivityPanel extends Component {
 	// TODO Pull in dynamic unread status/count
 	getTabs() {
 		return [
-			{
```
Comment (timmyc) [inline client/layout/activity-panel/index.js]: TTFN! But we will see you again soon inbox :wave:

**Review by timmyc — APPROVED** [review]
Looking good to me!

**Pull request merged**